samples/unit-tests/annotations/annotations-dynamic/demo.js

```diff
@@ -204,6 +204,22 @@ QUnit.test("Annotation's dynamic methods", function (assert) {
         'blue',
         '#15424: Fourth line should be blue (lineColor)'
     );
+
+    // TODO: Fails in firefox on CI
```

Comment:
We need to resolve this one before we merge

Reply:
I replaced it with just checking the class, we already have some `SVGLabel` hovering coverage in the button test.